Wrappers/Python/cil/plugins/tigre/ProjectionOperator.py

```diff
@@ -38,29 +38,61 @@
 class ProjectionOperator(LinearOperator):
     '''TIGRE Projection Operator'''
 
-    def __init__(self, image_geometry, aquisition_geometry, direct_method='interpolated',adjoint_weights='matched'):
-        '''
-        This class creates a configured TIGRE ProjectionOperator
-        
+    def __init__(self, image_geometry=None, acquisition_geometry=None, direct_method='interpolated',adjoint_weights='matched', **kwargs): 
+    
+        """
+        ProjectionOperator configures and calls TIGRE Projectors for your dataset.
+
         Please refer to the TIGRE documentation for futher descriptions
         https://github.com/CERN/TIGRE
         https://iopscience.iop.org/article/10.1088/2057-1976/2/5/055010
                         
-        :param image_geometry: A description of the ImageGeometry of your data
-        :type image_geometry: ImageGeometry
-        :param aquisition_geometry: A description of the AcquisitionGeometry of your data
-        :type aquisition_geometry: AcquisitionGeometry
-        :param direct_method: The method used by the forward projector, 'Siddon' for ray-voxel intersection, 'interpolated' for interpolated projection
-        :type direct_method: str, default 'interpolated'
-        :param adjoint_weights: The weighting method used by the cone-beam backward projector, 'matched' for weights to approximatly match the 'interpolated' forward projector, 'FDK' for FDK weights, default 'matched'
-        :type adjoint_weights: str    
-        '''
+
+        Parameters
+        ----------
+
+        image_geometry : ImageGeometry, default used if None
+            A description of the area/volume to reconstruct
+
+        acquisition_geometry : AcquisitionGeometry
+            A description of the acquisition data
+
+        direct_method: str,  default 'interpolated'
+            The method used by the forward projector, 'Siddon' for ray-voxel intersection, 'interpolated' for interpolated projection
+
+        adjoint_weights str, default 'matched'
```

Comment:
There's an issue with the display of the docs:

![image](https://user-images.githubusercontent.com/60604372/180239513-0b9589b7-65fa-4f4f-88e9-d89e937224aa.png)

Reply:
hopefully fixed, will check when it's rebuilt

Reply:
![image](https://user-images.githubusercontent.com/47746591/180264309-56c3879e-e373-4956-9a1e-016aa65b43cf.png)

Reply:
yay!